SpatialGDK/Documentation/content/unreal-editor-interface/editor-settings.md

```diff
@@ -0,0 +1,88 @@
+<%(TOC)%>
+# SpatialOS Editor Settings panel
+In the [Project Settings](https://docs.unrealengine.com/en-us/Engine/UI/ProjectSettings) editor, you can use the SpatialOS Editor Settings panel to change the default settings for local deployments, schema generation, and snapshot generation. For example, you might want to use the panel to extend the size of your [game world]({{urlRoot}}/content/glossary#game-world) or change your [load balancing]({{urlRoot}}/content/glossary#load-balancing) strategy.
+
+> **Note** : The SpatialOS editor settings are in the `DefaultSpatialGDKEditorSettings.ini file` in the `<GameRoot>\Config\ directory`, which you can edit with any text editor.
+
+## Access the panel
+To access the **SpatialOS Editor Settings** panel in the Unreal Editor, use either of the following methods:
+
+* From the **Edit** menu:
+	1. Select **Project Settings** to display the Project Settings editor.
+	1.  Scroll down to **SpatialOS GDK for Unreal** and select **Editor Settings** to display the SpatialOS Editor Settings panel.
+
+
+* From the Unreal Toolbar:
+	1. Select **Play** > **SpatialOS Settings** to display the Project Settings editor. 
+	1. Scroll down to **SpatialOS GDK for Unreal** and select **Editor Settings** to display the SpatialOS Editor Settings panel.
+
+  <img src="{{assetRoot}}assets/screen-grabs/editor-settings.png"/>
+    <br>_Image: the SpatialOS Editor Settings panel_
+
+## Settings
+The following table lists all the settings in the SpatialOS Editor Settings panel:
+
+<table>
+  <tr>
+    <th>Sections</th>
+    <th>Settings</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><strong>General</strong></td>
+    <td><strong>SpatialOS directory</strong></td>
+    <td>The directory for SpatialOS-related files, for example, `C:Projects/MyGame/spatial/.`</td>
```

Comment:
Backticks don't seem to work in `<td>` blocks. Use `<code>` instead.
```suggestion
    <td>The directory for SpatialOS-related files, for example, <code>C:Projects/MyGame/spatial/</code>.</td>
```

Reply:
Oh, my bad. Just remembered that this is supposed to be viewed on our docs page and not in GitHub markdown.